server/apps/main/views.py

```diff
@@ -116,11 +117,15 @@ def index(request):
     """
     if request.user.is_authenticated:
         oh_member = request.user.openhumansmember
+
+        stories = get_carousel_stories()
```

Comment:
For the laughs I'll put this here what I had written as the original top-level comment for the PR: 

> It might be because my brain is a bit fried, but I think I struggle to understand how the carousel bit is supposed to work in practice, in particular how it prioritizes what to show. In my local dev setup (using the `carousel.json` from this PR without any changes but having some approved local stories) the carousel is just completely empty. Is that the expected behaviour? 🙂 

Having had a second look, this bit is just put at the wrong place. Currently we only `get_carousel_stories()` for users that are authenticated, and that's why my logged-out session didn't load any stories 😂

Reply:
I should add: Logged-in the carousel works as I would kind of have expected, it shows the pre-existing data from the carousel.json

Reply:
Nice catch! This should be fixed now.